### PR TITLE
[wasm] Add blazor First UI measurement

### PR DESCRIPTION
**Pull request by radekdoulik**
This measures the time of `OnAfterRender` event on index page, to mimic the blazor benchmark's `First UI` measurement.

Also

 * fix blazor frame url base

 * let simple-server handle non-root urls with trailing '/'

### CODE REVIEW

File: src/mono/sample/wasm/blazor-frame/wwwroot/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <title>blazor</title>
-    <!-- <base href="/blazor-template/" /> -->
```
Comment (maraf) [inline src/mono/sample/wasm/blazor-frame/wwwroot/index.html:7]: Just curious, what's the reason for this change?

**Review by maraf — APPROVED** [review]
Looks good to me 👍

File: src/mono/sample/wasm/blazor-frame/wwwroot/index.html (hunk 1 above)
Comment (radekdoulik) [reply src/mono/sample/wasm/blazor-frame/wwwroot/index.html:7]: The `Index.razor` page wasn't loading properly, so I changed it according to https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/?view=aspnetcore-8.0&tabs=visual-studio#app-base-path

Interestingly I had to also change the `blazor.webassembly.js` path to make it work.
